### DISCUSSION TITLE
Installing requires 275MB and install firefox on ubuntu?

### DISCUSSION THREAD

**Discussion by ivanjaros** [libvips/libvips#4298] [Q&A] [answered]
I just ran `sudo apt install libvips-tools` on ubuntu server and it downloaded 275Mb of data, including firefox. What :D ?

Am i supposed to install different package if i just want to call vips cli?

Also it installed version 8.12.1. Last version i have on my windows is is 8.16.0(and is 40MB in total with all dlls).

PS: trying libvips instead of libvips-tools wants 325MB.

**Comment by lovell** [comment, marked as answer]
Please see https://github.com/libvips/libvips/discussions/2812 - this is a question for the maintainers of the libvips Debian package.

**Reply by ivanjaros** [reply]
Thanks. I am running Ubuntu Server 22.4.5 so when i'll be able to get on the latest version this should no longer ba problem.

**Reply by jcupitt** [reply]
Until then you can always just use `apt install libvips-tools --no-install-recommends`, of course.
